gen-apidocs/generators/util.go

```diff
@@ -84,74 +84,3 @@ func PrintInfo(config *api.Config) {
 	//	}
 	//}
 }
-
```

Comment:
Is PrintInfo(), JsonOutputFile(manifest.json) used?

Reply:
PrintInfo is still referenced, by JsonOutputFile is not (will fix).

Reply:
@tengqm , How do these changes affect the generation of the kubectl command reference?

```
docker run -v $(shell pwd)/gen-kubectldocs/generators/includes:/source -v $(shell pwd)/gen-kubectldocs/generators/build:/build -v $(shell pwd)/gen-kubectldocs/generators/:/manifest pwittrock/brodocs
```
Eventually the tooling/build of the kubectl reference will be updated (I hope to reopen the related kubectl reference generation PR and continue work) too.

Reply:
This change is irrelevant to the gen-kubectldocs. This PR is only related to the API reference generation. I'd like to have the kubectl reference reworked as well.